charts/management-portal/templates/configmap.yaml

```
@@ -8,6 +8,12 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 data:
+  healthcheck.sh: |
+    #!/bin/sh
+    STATUS=$(wget -O - localhost:8080/managementportal/management/health | grep -Fo 'db":{"status":"UP')
+    if [ -z "$(echo $STATUS)" ]; then
+      exit 1
+    fi
```

Alternative
```suggestion
    if ! (wget -O - localhost:8080/managementportal/management/health | grep -Fq 'db":{"status":"UP'); then
      exit 1
    fi
```

or even

```bash
    STATUS=$(wget -O - localhost:8080/managementportal/management/health)
    if ! (echo "$STATUS" | grep -Fq 'db":{"status":"UP'); then
      exit 1
    fi
```

---

Updated the code